### PR TITLE
Fix examples with wrong or empty html

### PR DESCRIPTION
**Pull request by tavriaforever**
// cc @gela-d @tormozz48 @tadatuta

### CODE REVIEW

File: src/blocks/common.blocks/block-jsdoc/__type-value/block-jsdoc__type-value.bemhtml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+block('block-jsdoc').elem('type-value')(
+    content()(function() {
+        return this.xmlEscape(applyNext());
+    })
+);
```
Comment (tadatuta) [inline src/blocks/common.blocks/block-jsdoc/__type-value/block-jsdoc__type-value.bemhtml]: new line at EOF

Comment (tavriaforever) [inline src/blocks/common.blocks/block-jsdoc/__type-value/block-jsdoc__type-value.bemhtml]: fix

File: src/blocks/common.blocks/block-jsdoc/__type/block-jsdoc__type.bemhtml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+block('block-jsdoc').elem('type')(
+    tag()('code')
+);
```
Comment (sipayRT) [inline src/blocks/common.blocks/block-jsdoc/__type/block-jsdoc__type.bemhtml]: new line at EOF

@gela-d @tavriaforever just check this option in your WebStorm config
webstorm

Comment (tavriaforever) [inline src/blocks/common.blocks/block-jsdoc/__type/block-jsdoc__type.bemhtml]: Fix
p.s. Thanks for tip!

File: src/blocks/common.blocks/block/block.bemtree [hunk 3]
```diff
@@ -15,17 +15,17 @@ block('block')(
             tabsHeader = [],
             tabsContent = [];
 
-        if(_data){
+        if(!_.isEmpty(_data)){
```
Comment (gela-d) [inline src/blocks/common.blocks/block/block.bemtree]: need space before `{`

**Comment by tormozz48** [comment]
:ok:

**Pull request merged**